plugins/CustomDimensions/tests/UI/CustomDimensions_spec.js

```diff
@@ -148,9 +148,6 @@ describe("CustomDimensions", function () {
             await (await page.jQuery('.dataTable .subDataTable .value:contains(en):first')).click();
             await page.waitForNetworkIdle();
             await page.waitForTimeout(500);
-            await page.evaluate(() => { // give table headers constant width so the screenshot stays the same
```

Comment:
This hopefully shouldn't be needed anymore, as the new resizing logic should be more solid.